pkg/controller/pod.go

```diff
@@ -1297,7 +1297,7 @@ func (c *Controller) acquireAddress(pod *v1.Pod, podNet *kubeovnNet) (string, st
 		for _, net := range nsNets {
 			for _, staticIPs := range ipPool {
 				for _, staticIP := range strings.Split(staticIPs, ",") {
-					if c.ipam.IsIPAssignedToPod(staticIP, net.Subnet.Name, key) {
+					if c.ipam.IsIPAssignedToOtherPod(staticIP, net.Subnet.Name, key) {
 						klog.Errorf("static address %s for %s has been assigned", staticIP, key)
 						continue
```

Comment:
Can we provide information about the pod to which the IP is assigned? This is helpful for debugging.

Reply:
got it

Reply:
subnet.isIPAssignedToOtherPod provide the log:

		klog.V(4).Infof("v4 check ip assigned, existPod %s, podName %s", existPod, podName)

existedPod is  the pod to which the IP is assigned

Reply:
`klog.V(4).Infof` means the message will be printed only when the logging verbosity level is greater than or equal to 4.

To change the verbosity level from the default value to 4, we need to add an additional command argument`-v=4` to kube-ovn-controller.

Reply:
Do you mean below?

klog.V(4).Infof  => klog.Infof

Reply:
I'd prefer this:

```go
if assignedPod, ok := c.ipam.IsIPAssignedToOtherPod(staticIP, net.Subnet.Name, key); ok {
    klog.Errorf("static address %s for %s has been assigned to %s", staticIP, key, assignedPod)
    // ...
}
```

Reply:
ok, I push my new pr